javaslang-benchmark/src/test/java/javaslang/JmhRunner.java

```diff
@@ -112,4 +114,28 @@ public static void runAndReport(Array<Class<?>> groups, int warmupIterations, in
     public static int aggregate(int x, int y) {
         return x ^ y;
     }
+
+    /** speeds up construction of immutable objects by caching them */
+    private static Map<Tuple2<Integer, Function1<?, ?>>, Object> FUNCTION_CACHE = HashMap.empty();
```

Comment:
I think we do not need the cache. However, the create() method looks like a simplification. We could use it without caching. What do you think?